src/app/[bookId]/components/book-info.tsx

```diff
@@ -0,0 +1,77 @@
+'use client'
+
+import { Star } from 'lucide-react'
```

Comment:
```suggestion
import { Star } from 'lucide-react'
import Image from 'next/image'
```
import Image components from next

Reply:
Fixed it